person.rb

```diff
@@ -1,5 +1,5 @@
-require 'corrector'
-require 'rental'
+require './corrector'
+require './rental'
 
 class Person
   attr_accessor :name, :age, :rentals
```

Comment:
- [ ] I was also unable to create a teacher.

![image](https://user-images.githubusercontent.com/16058470/136662261-769cf9c8-56c9-4bd0-8961-08eb7489b9fc.png)

Kindly see to it.